frontend/src/views/Categories.js

```diff
@@ -27,7 +42,7 @@ const Categories = () => {
       flexWrap="wrap"
       paddingX="1rem"
     >
-      {state.categories.map((category) => (
+      {sortedCategories.map((category) => (
         <Pressable
           onPress={() => {
             state.category = category;
```

Comment:
The code patch looks good. 

Two improvements that could be suggested:

1. Sorting the categories array in place using Array.prototype.sort() method; no need to create a deep copy of the categories array using JSON.stringify() and then parse it with JSON.parse(). This is more efficient.

2. Using Object destructuring, `const state = globalState();` could be replaced with `const {categories, category} = globalState();` since only 'categories' and 'category' properties are used from the 'state' object. This is more concise.